engine/runtime/src/main/java/org/enso/interpreter/runtime/scope/ModuleScope.java

```diff
@@ -295,4 +307,35 @@ public void reset() {
     conversions = new HashMap<>();
     polyglotSymbols = new HashMap<>();
   }
+
+  public boolean simpleUpdate(model.TextEdit edit) {
```

Comment:
probably need to merge this with `Module` soon. This method would look awkward in either currently